sdk/securitydevops/Azure.ResourceManager.SecurityDevOps/Samples/Azure.ResourceManager.SecurityDevOps.Samples.csproj

```diff
@@ -1,10 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This is referenced in the solution despite itself got removed

Reply:
it is a mistake, I will revert this removal